src/classes/LineItemTriggerHandler.cls

```diff
@@ -12,6 +12,7 @@ public with sharing class LineItemTriggerHandler {
 		Set<Id> orderIds = new Set<Id>();
 		for (Line_Item__c i : items) {
 			orderIds.add(i.Order__c);
+            update i;
```

Comment:
This DML statement is inside a loop. Use a collection to store relevant data, and move the DML statement outside the loop.